rollup.config.js

```diff
@@ -1,11 +1,10 @@
+// @ts-check
```

Comment:
can this file be TS?

Reply:
it requires installing a whole add'l dependency so i opted not to: https://rollupjs.org/command-line-interface/#config-intellisense